ui/components/ui/box/box.js

```diff
@@ -12,26 +12,11 @@ import {
   TEXT_ALIGN,
   FLEX_DIRECTION,
   FLEX_WRAP,
+  BREAKPOINTS,
```

Comment:
Importing new `BREAKPOINTS` const here. It contains the most commonly used breakpoints in the codebase. I have another PR standardizing media queries that can be reviewed separately https://github.com/MetaMask/metamask-extension/pull/15053